<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-1RBF983" failures="1" tests="1" name="Test" time="0.921" errors="0" timestamp="2022-10-11T21:30:26 BST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.311">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.dbank.stepdefinitions.StepDefinitions.i_GET_the_details_for_NACE(StepDefinitions.java:16)
at âœ½.I GET the details for NACE "398481"(file:///C:/Projects/RestAssuredAutomationFramework/src/main/resources/features/naceListValidation/NaceDetails.feature:5)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Test -->
